tests/test_pipenv.py

```diff
@@ -878,6 +878,23 @@ def test_activate_virtualenv_no_source(self):
 
         assert command == '{0}/bin/activate'.format(venv)
 
+    @pytest.mark.lock
+    def test_lock_ignore_eggs(self, pypi):
+        """Ensure locking works with packages provoding egg formats.
+
+        See https://github.com/pypa/pipenv/issues/1849.
+        TODO: This hits the real PyPI. What happens if RandomWords somehow
+        changes its uploads? Need a better way to test this.
```

Comment:
WE mock the rest of pypi, just mock it (or I can if you don't mind me pushing to this branch)

Reply:
I’m not sure what I need to do here, so sure, go for it.